modules/core/src/main/scala/fs2/kafka/Header.scala

```diff
@@ -62,4 +66,10 @@ object Header {
 
   implicit val headerShow: Show[Header] =
     Show.fromToString
+
+  implicit val headerEq: Eq[Header] =
+    Eq.instance {
+      case (l, r) =>
+        l.key === r.key && l.value.toVector === r.value.toVector
```

Comment:
Since there is no `Eq[Array[Byte]]` I converted arrays to a vector.

Reply:
I thought that converting to vector will have a too big performance impact. Now I'm using `sameElements`.